packages/safe-ds-eda/types/state.d.ts

```diff
@@ -77,26 +77,25 @@ export interface Profiling {
 
 export interface ProfilingDetailBase {
     type: 'numerical' | 'image' | 'name';
-    name: string;
+    value: string;
+    interpretation: 'warn' | 'error' | 'default' | 'bold' | 'good';
```

Comment:
Is `interpretation` relevant to all subtypes of `ProfilingDetailBase`, including `ProfilingDetailImage`? If not, better add an intermediate interface like `ProfilingDetailsText`.